src/Wallabag/CoreBundle/Resources/views/Entry/entries.xml.twig

```diff
@@ -27,7 +27,7 @@
         <name>{{ user }}</name>
     </author>
     <icon>{{ asset('favicon.ico') }}</icon>
-    <logo>{{ asset('bundles/wallabagcore/themes/_global/img/logo-square.png') }}</logo>
+    <logo>{{ asset('img/logo-square.png') }}</logo>
```

Comment:
not really sure about this one to be honest, but it looks it was broken before as there is no `themes/_global/img/logo-square.png` in CoreBundle